charts/zookeeper-operator/values.yaml

```diff
@@ -47,3 +47,18 @@ affinity: {}
 tolerations: []
 # Pod annotations
 annotations: {}
+
+## AdditionalSidecar configuration details.
+additionalSidecars: {}
+# - name: nginx
+#   image: nginx:latest
+
+## AdditionalSidecarEnvironment configuration details.
+additionalSidecarEnv: {}
+
+## AdditionalSidecarVolumes configuration details.
+additionalSidecarVolumes: {}
```

Comment:
```suggestion
additionalVolumes: {}
```

Reply:
Renamed the additionalSidecarVolumes to additionalVolumes.